app/controllers/api/v1/admin/accounts_controller.rb

```diff
@@ -44,6 +44,45 @@ def show
     render json: @account, serializer: REST::Admin::AccountSerializer
   end
 
+  def create
+    account = Account.new(username: params[:username])
+
+    user = User.new(
+      email: params[:email],
+      password: params[:password],
+      agreement: true,
+      approved: true,
+      admin: params[:role] == 'admin',
+      moderator: params[:role] == 'moderator',
+      confirmed_at: params[:confirmed] ? Time.now.utc : nil,
+      bypass_invite_request_check: true
+    )
+
+    if params[:reattach]
+      account = Account.find_local(params[:username]) || account
+
+      if account.user.present? && !params[:force]
+        render json: { error: 'The chosen username is currently in use. Use `force: true` to reattach it anyway and delete the other user.' }, status: 422
+      elsif account.user.present?
+        DeleteAccountService.new.call(account, reserve_email: false)
+      end
+    end
```

Comment:
I'm not sure we should support these very specific use cases from the admin API, but I have no strong opinion either way.